lib/utils.ts

```diff
@@ -4,3 +4,7 @@ import { twMerge } from "tailwind-merge"
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs))
 }
+
+export const typedObjectEntries = <TKey extends string, TValue>(input: Record<TKey, TValue>): [TKey, TValue][] => {
```

Comment:
Default return type is `[string, TValue]` which isn't ideal for me. `[TKey, TValue]` is more strict and accurate.